tests/lint/readme.md

```diff
@@ -4,6 +4,72 @@ Our linting process with Flake8 consists of two main steps:
 - Generating a diff
 - Running Flake8 on the diff
 
+## Common problems
+There seem to be several common situations for which the Flake8 errors don't clearly indicate an error free solution.
+
+### Line-breaking functions and statements
+
+Lint errors you may encounter:
+- ET128 (flake8-tabs) *unexpected number of tabs and spaces at start of expression line*
+  - In particular, if this error message "expects x spaces", its likely that it is expecting a
+  code alignment style, rather than hanging indent style.
+  To change this expectation, first ensure that there is a newline after the opening
+  paren/bracket/brace.
+- ET121 (flake8-tabs) *unexpected number of tabs at start of definition line*
+
+#### Preferred formatting
+
+- Break after the parenthesis, putting the first parameter on a new line.
+- Double indent the params to avoid ET121
+
+```python
+# method with many parameters
+# start params on new line to avoid aligning with parenthesis.
+def foo(
+		arg1,  # double indent to avoid ET121
+		arg2
+):
+	# long expression
+	# start params on new line to avoid aligning with parenthesis.
+	if(
+		arg1 is not None

```

Comment:
```suggestion
		arg1 is not None  # NO concerns about ET121
```

See also my previous comment. It should be clear from the readme that double indentation is necessary in the definition case but not in the expression case.